src/public/app/widgets/dialogs/branch_prefix.js

```diff
@@ -14,24 +14,17 @@ const TPL = `<div class="branch-prefix-dialog modal fade mx-auto" tabindex="-1"
         <form class="branch-prefix-form">
             <div class="modal-content">
                 <div class="modal-header">
-                    <h5 class="modal-title mr-auto">${t('branch_prefix.edit_branch_prefix')}</h5>
-
+                    <h5 class="modal-title flex-grow-1">${t('branch_prefix.edit_branch_prefix')}</h5>
                     <button class="help-button" type="button" data-help-page="tree-concepts.html#prefix" title="${t('branch_prefix.help_on_tree_prefix')}">?</button>
-
-                    <button type="button" class="close" data-dismiss="modal" aria-label="${t('branch_prefix.close')}" style="margin-left: 0;">
-                        <span aria-hidden="true">&times;</span>
-                    </button>
+                    <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="${t('branch_prefix.close')}"></button>
```

Comment:
<img width="785" alt="image" src="https://github.com/user-attachments/assets/46a6f322-2c86-495b-ac27-1397b6f509fe">

Background regression for input and maybe the color of the X button?